### PR TITLE
release: v5.2.0

### PR DESCRIPTION
**Pull request by omaralbeik**

### CODE REVIEW

File: Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift [hunk 1]
```diff
@@ -122,7 +122,7 @@ public extension Array where Element: Equatable {
     /// - Returns: an array of unique elements.
     func withoutDuplicates<E: Equatable>(keyPath path: KeyPath<Element, E>) -> [Element] {
         return reduce(into: [Element]()) { (result, element) in
-            if !result.contains { $0[keyPath: path] == element[keyPath: path] } {
+            if !result.contains(where: { $0[keyPath: path] == element[keyPath: path] }) {
```
Comment (guykogus) [inline Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift:125]: Why this change?

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@
```
Comment (guykogus) [inline README.md:17]: I can't select it, but a few lines up there's the Xcode version. Should we mark that as Xcode 11.4?

File: Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift (hunk 1 above)
Comment (omaralbeik) [reply Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift:125]: SwiftLint complains if you have nested trailing closures

**Comment by LucianoPAlmeida** [comment]
Does the CI already support the new Xcode 11.4? If it does we could open an issue `good first time` to update it together with Package.swift tools to 5.2 :)

**Comment by omaralbeik** [comment]
> Actually will you update `.github/workflows/CI.yml`?

Looks like Github Actions does not support 11.4 yet
https://github.com/Alamofire/Alamofire/pull/3134
